sdks/java/build-tools/pom.xml

```diff
@@ -28,5 +28,34 @@
 
   <artifactId>beam-sdks-java-build-tools</artifactId>
   <name>Apache Beam :: SDKs :: Java :: Build Tools</name>
-  
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>net.revelc.code.formatter</groupId>
+        <artifactId>formatter-maven-plugin</artifactId>
+        <version>2.0.1</version>
+        <executions>
+          <execution>
+            <id>test-code-style</id>
+            <phase>process-test-sources</phase>
+            <goals>
+              <goal>format</goal>
+            </goals>
+            <configuration>
+              <configFile>${project.basedir}/src/main/resources/beam/beam-codestyle.xml</configFile>
+              <directories>
+                <directory>${project.build.testSourceDirectory}</directory>
+              </directories>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
`build-tools` parent is `beam-parent`. So `maven-checkstyle-plugin` is inherited in `pluginsManagement` but not in `plugins`.